src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.c

```diff
@@ -27,17 +26,47 @@ const ActorInit Bg_Inibs_Movebg_InitVars = {
     (ActorFunc)BgInibsMovebg_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B96578[] = {
+Gfx* D_80B96560[] = { 0x060062D8, 0x06001DC0 };
+Gfx* D_80B96568[] = { 0x06006140, 0x06001C10 };
+AnimatedMaterial* D_80B96570[] = { 0x06006858, 0x06002598 };
```

Comment:
This pointers are segmented pointers. Those refer to assets of this actor's object (you can tell by the `0x06` part of the pointer)
We like to use `extern`ed symbols for those
```suggestion
extern Gfx D_060062D8[];
extern Gfx D_06001DC0[];
extern Gfx D_06006140[];
extern Gfx D_06001C10[];
extern AnimatedMaterial D_06006858;
extern AnimatedMaterial D_06002598;

Gfx* D_80B96560[] = { D_060062D8, D_06001DC0 };
Gfx* D_80B96568[] = { D_06006140, D_06001C10 };
AnimatedMaterial* D_80B96570[] = { &D_06006858, &D_06002598 };
```

You'll need to add those to undefined_syms.txt too